app/tests/unit/TestTest.php

```diff
@@ -1,23 +1,203 @@
 <?php
 
-declare(strict_types = 1);
+declare(strict_types=1);
 
 namespace Tests\unit;
 
 use PHPUnit\Framework\TestCase;
+use SocialPost\Dto\SocialPostTo;
+use Statistics\Dto\ParamsTo;
+use Statistics\Calculator\AveragePostsByUsersPerMonth;
 
 /**
  * Class ATestTest
  *
  * @package Tests\unit
  */
-class TestTest extends TestCase
+class AveragePostsByUsersPerMonthTest extends TestCase
 {
-    /**
-     * @test
-     */
-    public function testNothing(): void
+    // we don't test this here
+    // I would have tested dated in the abstract calculator
+    private $startDate;
```

Comment:
No reason to test date stuff here.
That should be done in the `AbstractCalculator`.

Reply:
Besides the way filtering done is quite horrendous =)
We still ask everything from the API and filter in code.
I did not touch that as this was not in scope of the exercise.